packages/alfa-network/package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "$schema": "http://json.schemastore.org/package",
+  "name": "@siteimprove/alfa-network",
+  "homepage": "https://siteimprove.com",
+  "version": "0.10.0",
+  "license": "MIT",
+  "description": "An implementation of an immutable, directed graph that allows for multiple, unique edges",
```

Comment:
Isn't that a (directed) multigraph?
(which is apparently also called a quiver 🤷 )
https://en.wikipedia.org/wiki/Multigraph#Directed_multigraph_(edges_with_own_identity)
https://en.wikipedia.org/wiki/Quiver_(mathematics)

Reply:
That's exactly what it is, yes 👍 I named it "network" to align with Google Guava (https://guava.dev/releases/snapshot/api/docs/com/google/common/graph/Network.html). Didn't know about the name "quiver" though, that's neat 😄